.github/workflows/manpage.yml

```diff
@@ -0,0 +1,34 @@
+name: Building man page and man.test
+on:
+  push:
+    paths:
+      - '.github/workflows/manpage.yml'
+      - 'docs/**'
+      - 'tests/test.man'
+
+jobs:
+  manpages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.11'
+          cache: pipenv
+      - name: Install pipenv
+        run: pip install pipenv
+      - name: Install dependencies
+        run: pipenv sync
+        working-directory: docs
+      - name: Configure
+        run: |
+          autoreconf -i
+          ./configure --enable-docs
+      - name: Build man page and man.test
+        run: |
+          rm -f jq.1.prebuilt tests/man.test
+          make jq.1.prebuilt tests/man.test
+      - name: Make sure man.test is up to date
+        run: git diff --exit-code tests/man.test
```

Comment:
Why not check jq.1.prebuilt as well?

Reply:
Because `jq.1.prebuilt` contains the build date (first of the month) on its second line:

```roff
.
.TH "JQ" "1" "July 2023" "" ""
```

So it will start failing next month if we don't update `manual.yml`, but update some other file in `docs/` in the mean time.

Maybe that date could be removed to allow checking if it is up to date here

Reply:
I saved the original `jq.1.prebuilt`, and then compared the newly generated one with it ignoring the first two lines using.
```sh
test -s jq.1.prebuilt
diff -- <(tail -n +3 jq.1.old) <(tail -n +3 jq.1.prebuilt)
```